lib/bourne-shell.sh

```diff
@@ -3,7 +3,13 @@
 # see /usr/share/doc/bash/examples/startup-files (in the package bash-doc)
 
 # make less more friendly for non-text input files, see lesspipe(1)
-[ -x /usr/bin/lesspipe ] && eval "$(SHELL=/bin/sh lesspipe)"
+if [ -z "${LESSOPEN}" ] && [ ! -f /etc/gentoo-release ]; then
+  if [ -x /usr/bin/lesspipe ]; then
+    eval "$(SHELL=/bin/sh lesspipe)"
```

Comment:
How about making it consistent with the check on the previous line? The same for `lesspipe.sh`.

```suggestion
    eval "$(SHELL=/bin/sh /usr/bin/lesspipe)"
```